cds_ils/ldap/api.py

```diff
@@ -270,26 +272,43 @@ def delete_users(dry_run=True):
     # get all Invenio remote accounts and prepare a list with needed info
     invenio_users = remap_invenio_users(log_func)
 
-    for invenio_user in invenio_users:
+    for remote_account in invenio_users:
+        try:
+            invenio_user = InvenioUser(remote_account)
+        except NoResultFound:
+            continue
+
         ldap_user = ldap_users_map.get(
-            invenio_user["remote_account_person_id"]
-        )
+                invenio_user.data["remote_account_person_id"]
```

Comment:
Oh wow, did we miss this `data`?

Reply:
it was due to refactoring in the sync itself, and this method was overlooked in the refactoring, since it was not used